core/src/main/java/org/apache/carbondata/core/datastore/page/ColumnPage.java

```diff
@@ -68,8 +74,38 @@ public DataType getDataType() {
     return dataType;
   }
 
-  public Object getStatistics() {
-    return statsCollector.getPageStats();
+  public SimpleStatsResult getStatistics() {
+    if (statsCollector != null) {
+      return statsCollector.getPageStats();
+    } else {
+      // return a dummy result, for complex column
+      return new SimpleStatsResult() {
```

Comment:
Better create class separately and set to ColumnPage instead of creating class instance every time.